### ISSUE TITLE
Low volume mumble voices even at 200% volume on headphones

### ISSUE THREAD

**Issue by Anduviel** [mumble-voip/mumble#3526]
I recently got a new desktop with all modern configurations and realtek sound on board, almost the same a previous desktop where I had no issues like that at all.
I have set up mumble, and have issues with what later turned out to be low volume from the headphones.
The headphones (Steelseries Siberia 200) play other sounds and music loud, no problems. 
Mumble sound on the speakers is also fine
People here me speaking loud and clear on mumble, so microphone is ok. 
So it is the combination of headphones and mumble

I tried to put it on 200% but then I can only hear that someone is speaking and not what they are saying.

I have issues isolating the problem and I wonder if some may experienced the same and / or have a solution for this problem.

Using default setting exept for volume, and I know from the other sounds that I am using the proper device on the list.

Installed new realtek drivers didnt help.

Thanks in advance.

**Comment by nikolaenko20000-del** [comment]
I recently ran into the same issue with my Sennheiser HD 280 Pro headphones—I join a channel and can barely hear anyone. I checked all the settings and all the devices I’m using—everything is the same as in other apps, like Teamspeak or Discord. Boosting the volume by 200% didn’t help; only increasing the server volume by +18 dB did. Reinstalling the app didn’t help either. I’d appreciate any suggestions for a possible solution.
